src/EditorFeatures/Test2/InlineParameterNameHints/AbstractInlineParameterNameHintsTests.vb

```diff
@@ -25,7 +25,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.InlineParameterNameHints
                 Dim hostDocument = workspace.Documents.Single()
                 Dim snapshot = hostDocument.GetTextBuffer().CurrentSnapshot
                 Dim document = workspace.CurrentSolution.GetDocument(hostDocument.Id)
-                Dim tagService = document.GetRequiredLanguageService(Of IInlineParameterNameHintsService)
+                Dim tagService = document.GetLanguageService(Of IInlineParameterNameHintsService)
```

Comment:
Why the change?

Reply:
So I forgot to change it back. I had thought that was the reason my tests weren't running, but I had not changed the language type for each test itself to VB.